apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/options/CreateUserOptions.java

```diff
@@ -97,7 +97,7 @@ private ServerRequest(String name, String password) {
       if (password != null)
          user.password = password;
       if (tenant != null)
-         user.tenant = tenant;
+         user.tenantId = tenant;
```

Comment:
Would it make sense to also rename the other variable to `tenantId`, or should we leave that as it is?